<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramPageSchedule</class>
 <widget class="QWidget" name="ProgramPageSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Start</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QComboBox" name="combo_start_condition">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Never</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Now</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Reboot</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Time</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Condition:</string>
          </property>
          <property name="buddy">
           <cstring>combo_start_condition</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_start_time">
          <property name="text">
           <string>Time:</string>
          </property>
          <property name="buddy">
           <cstring>date_start_time</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateTimeEdit" name="date_start_time">
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1970</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1770</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>dd.MM.yyyy HH:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_start_delay">
          <property name="text">
           <string>Delay:</string>
          </property>
          <property name="buddy">
           <cstring>spin_start_delay</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spin_start_delay">
          <property name="suffix">
           <string> seconds</string>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_start_condition_never_help">
          <property name="text">
           <string>The program will never be executed.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_start_condition_now_help">
          <property name="text">
           <string>The program will be executed after the specified delay has passed starting from now.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_start_condition_reboot_help">
          <property name="text">
           <string>The program will be executed after the specified delay has passed starting from every reboot.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_start_condition_time_help">
          <property name="text">
           <string>The program will be executed at the specified time.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Repeat</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QComboBox" name="combo_repeat_mode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Never</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Interval</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cron</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Mode:</string>
          </property>
          <property name="buddy">
           <cstring>combo_repeat_mode</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_repeat_interval">
          <property name="text">
           <string>Inverval:</string>
          </property>
          <property name="buddy">
           <cstring>spin_repeat_interval</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_repeat_fields">
          <property name="text">
           <string>Fields:</string>
          </property>
          <property name="buddy">
           <cstring>edit_repeat_fields</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spin_repeat_interval">
          <property name="suffix">
           <string> seconds</string>
          </property>
          <property name="maximum">
           <number>2147483647</number>
          </property>
          <property name="value">
           <number>60</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="edit_repeat_fields">
          <property name="text">
           <string>* * * * *</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_repeat_mode_never_help">
          <property name="text">
           <string>The program will never be executed again after the first execution.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_repeat_mode_interval_help">
          <property name="text">
           <string>The program will be executed with the specified interval after the first execution.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_repeat_mode_cron_help">
          <property name="text">
           <string>The program will be executed if the current time matches the cron specification.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combo_start_condition</tabstop>
  <tabstop>date_start_time</tabstop>
  <tabstop>spin_start_delay</tabstop>
  <tabstop>combo_repeat_mode</tabstop>
  <tabstop>spin_repeat_interval</tabstop>
  <tabstop>edit_repeat_fields</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
